tutorial/creating_packages/other_types_of_packages/package_prebuilt_binaries.rst

```diff
@@ -1,2 +1,414 @@
-Package prebuild binaries
+Package prebuilt binaries
 =========================
+
+There are specific scenarios in which it is necessary to create packages from existing binaries, for example from 3rd
+parties or binaries previously built by another process or team that is not using Conan. Under these circumstances,
+building from sources is not what you want.
+
+You can package the local files in the following scenarios:
+
+ 1. When you are developing your package locally and you want to export the built artifacts to the local
+    cache. As you don't want to rebuild again (clean copy) your artifacts, you don't want to call
+    :command:`conan create`. This method will keep your build cache if you are using an IDE.
+ 2. When you cannot build the packages from sources (when only pre-built binaries are available) and you have them
+    in a local directory.
+ 3. Same as 2 but you have the precompiled libraries in a remote repository.
+
+
+Locally building binaries
+-------------------------
+
+This is an example of scenario 1 explained in the introduction. In this case, you can run :command:`conan export-pkg`
+command directly.
+
+Please, first clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
```

Comment:
Isn't this example reproducible from the ``conan new`` template? In theory it should, and things would be a bit simpler?

Reply:
If you change the `conanfile.py` it is.

Reply:
I've changed it. Please review